migrations/2023-10-24-030352_change_primary_keys_and_remove_some_id_columns/up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE post_saved
+    DROP COLUMN id,
+    ADD PRIMARY KEY (post_id, person_id),
```

Comment:
I think this index would be more efficient backwards? person_id as the first index element means it's quick to load all your saved posts, but maybe there's already an index for (person_id, published)

Reply:
There's already an index on just person_id.

I changed the primary key to (person_id, post_id) and removed the old person_id index so now there's only 1 index.